src/content/contribute/writers-guide.md

```diff
@@ -1,6 +1,12 @@
 ---
 title: Writer's Guide
 sort: 2
+contributors:
```

Comment:
Only add yourself to contributors list, some people choose not to appear on it.

Reply:
I can understand for @lhoff and @carolstran .
@skipjack and @byzyk , can you help me understand is there anything which I'm missing about this page that contribs don't want to add their name, as your name is there on other pages?

Reply:
@hemal7735 I touched almost every page in this site when I was maintaining it a while back, so I only added my name to ones that I wrote or made significant changes to. I appreciate you adding me but I agree with @montogeek that some folks may not want to be mentioned so it’s better to just add yourself. (and no aside from a desire to remain anonymous there’s no reason not to add yourself)